Sources/SharedServices/DynamoUserRepository.swift

```diff
@@ -39,7 +39,7 @@ struct DynamoUserRepository {
         let query = DynamoDB.QueryInput(
             expressionAttributeValues: [":v1": .s("DISCORD-\(id)")],
             indexName: discordIndex,
-            keyConditionExpression: "data1 = :v1",
+            keyConditionExpression: "discordID = :v1",
```

Comment:
You can't change `data1` here. This is a db query and there is no such column in the db as `discordID`.

Reply:
I thought we wanted to rename these two fields? I know they are not renamed in the db yet, but it’s a starting point

Reply:
yeah believe me when i say i'm not satisfied with these names 😅, but we probably should let that be for another time/PR since @0xTim wants to rework the db too.